internal/operators/lvm/lvm_operator.go

```diff
@@ -173,9 +173,9 @@ func (o *operator) GetPreflightRequirements(context context.Context, cluster *co
 		Requirements: &models.HostTypeHardwareRequirementsWrapper{
 			Master: &models.HostTypeHardwareRequirements{
 				Qualitative: []string{
-					"At least 1 non-boot disk on one or more host",
-					fmt.Sprintf("%d GiB of additional RAM", memoryRequirements),
```

Comment:
nit: maybe should specify MB in the name of the variable? i.e. `memoryRequirementsMB` or something?